lib/packages/shared-types/opensearch/main/index.ts

```diff
@@ -45,7 +46,7 @@ export type Document = z.infer<capitatedAmendment.Schema> &
     makoChangedDate: string;
     changelog?: Changelog[];
     appkChildren?: OmitFoundItemResult[];
-  };
+  } & z.infer<appK.Schema>;
```

Comment:
Nitpick: kinda an eyesore this isn't consistent like the rest of the schema imports, defined before the object

Reply:
i agree!